ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Ledger/Block.hs

```diff
@@ -337,9 +354,16 @@ data SimpleLedgerConfig c ext = SimpleLedgerConfig {
 deriving instance Show (MockLedgerConfig c ext) => Show (SimpleLedgerConfig c ext)
 deriving instance NoThunks (MockLedgerConfig c ext)
                => NoThunks (SimpleLedgerConfig c ext)
+deriving instance Serialise (MockLedgerConfig c ext)
+               => Serialise (SimpleLedgerConfig c ext)
 
 type instance LedgerCfg (LedgerState (SimpleBlock c ext)) = SimpleLedgerConfig c ext
 
+instance MockProtocolSpecific c ext => HasPartialLedgerConfig (SimpleBlock c ext)
+
+instance (Serialise (MockLedgerConfig c ext))
```

Comment:
Oof. It's a bummer that the `SimpleLedgerConfig` includes a `MockLedgerConfig` type family whereas the `MockBlock` is a synonym for a specialization of `SimpleBlock`; that naming seems upside-down :(

Reply:
Yes.. It's a bit odd. Can we address this as a separate issue?

Reply:
Right, yes. I was just sharing a grumble/sympathy, not requesting any action.